examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -226,16 +217,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "../../tests/resources/deeprec/sasrec/reviews_Electronics_5.txt\n",
-      "36262 Users and 35074 items\n",
-      "average sequence length: 15.01\n"
+      "20247 Users and 11589 items\n",
```

Comment:
Why do we have now fewer users and items? Did the data change?

Reply:
The source data did not change. I think it was some intermediate file that was loaded earlier while I was testing with the <filter_K_core> function.

Reply:
So are the current numbers correct?

Reply:
do you know what is happening with the intermediate file?

Reply:
The intermediate files are stored in the data directory. 

I think this is correct and same as what we get from daisyRec package ([link](https://github.com/AmazingDD/daisyRec)). 

![image](https://user-images.githubusercontent.com/6950343/159691432-2ec8ac32-bdfa-426a-b16a-d764d358d92b.png)